src/boot/bootctl.c

```diff
@@ -1132,6 +1181,18 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_graceful = true;
                         break;
 
+                case ARG_MAKE_MID:
+                        if (!optarg || !strncmp(optarg, "yes", strlen("yes")))
+                                arg_make_mid = MID_YES;
+                        else if (!strncmp(optarg, "no", strlen("no")))
+                                arg_make_mid = MID_NO;
+                        else if (!strncmp(optarg, "auto", strlen("auto")))
+                                arg_make_mid = MID_AUTO;  /* default */
+                        else
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                                       "--make-machine-id-directory takes nothing/yes/no/auto");
```

Comment:
we usually suffix cmdline switches that take a value with "=" to indicate that, in log messages and in the man pages. i.e. please spell this out as:

```
"--make-machine-id-directory= takes …```
```

And I'd drop the "nothing" thing

Reply:
Fair, applied.